core/foundation/test/testNotFn.cxx

```diff
@@ -1,8 +1,3 @@
-// Including functional for __cpp_lib_not_fn
-#include <functional>
-
-#ifndef __cpp_lib_not_fn
```

Comment:
There is no point in us testing the stdlib implementation of not_fn.

Reply:
This does verify that the tests are correct, though. Okay, I'll fix.